src/hooks/useApi.ts

```
@@ -0,0 +1,140 @@
+    });
+
+    if (!response.ok) {
+      throw new Error("Failed to refresh token");
+    }
+
+    const data = await response.json();
+    localStorage.setItem("accessToken", data.access);
+    return data.access;
+  };
+
+  const apiCall = useCallback(
```

Biome (1.9.4)

[error] This hook does not specify all of its dependencies: refreshAccessToken

This dependency is not specified in the hook dependency list.


(lint/correctness/useExhaustiveDependencies)